CHANGES.md

```diff
@@ -1,10 +1,47 @@
+## edge-19.10.5
+
+This edge release adds support for integrating Linkerd's PKI with an external
+certificate issuer such as [`cert-manager`], adds distributed tracing support to
```

Comment:
@grampelberg Any thoughts on whether we want to call out control plane distributed tracing here? Currently, the CLI flag is hidden, for dev/debugging purposes only. I wonder if we should either leave it out or label it as _internal_.

Reply:
My vote is internal, we should tell folks you can do it ... just not encourage it.

Reply:
@grampelberg in that case, should it be removed from the release summary?